Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.3.0-DEV"
 
 [deps]
 AbstractFFTs = "621f4979-c628-5d54-868e-fcf4e3e8185c"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
```

Comment:
Maybe it's clear once I read through the changes below, but why do we need Compat?

Reply:
For `@constprop`, which was added in v1.7.